### DISCUSSION TITLE
Api file size limit?

### DISCUSSION THREAD

**Discussion by diodenschein** [inventree/InvenTree#4655] [Q&A] [answered]
I run into trouble when trying to upload pictures to parts. 
Mostly from iOS which really loves pictures bigger than 2mb. 

Can I (mostly where) increase the limit?

Same for attachments. 

I currently run into 413 error regularly. 

![image](https://user-images.githubusercontent.com/1050465/233674415-19a4196a-10e4-4acd-8729-2c98fc3a41c2.jpeg)

**Comment by SchrodingersGat** [comment, marked as answer]
@diodenschein that looks like an issue with your specific server setup - namely your nginx proxy is configured to restrict file size upload to 2mb.

**Reply by diodenschein** [reply]
Yes indeed, my bad. 

For anyone who comes here with similar problems: 

client_max_body_size 15M 

The default is actually 1M
